aws-code-build/ci/buildspec.yaml

```diff
@@ -0,0 +1,36 @@
+version: 0.2
+
+# ---- AUTOMATICALLY GENERATED FILE --------
+# ---- DO NOT EDIT MANUALLY, BUT USE PYTHON MODULE "script-languages-container-ci-setup" TO UPDATE ---
+
+batch:
+  fast-fail: false
+  build-graph:
+    - identifier: build_python_36_data_science_cuda_EXASOL_620
+      env:
+        variables:
+          FLAVOR: python-3.6-data-science-cuda-EXASOL-6.2.0
+      buildspec: ./aws-code-build/ci/build_buildspec.yaml
+      privileged-mode: true
+      type: BUILD_GENERAL1_MEDIUM
+    - identifier: build_python_36_minimal_EXASOL_620
+      env:
+        variables:
+          FLAVOR: python-3.6-minimal-EXASOL-6.2.0
+      buildspec: ./aws-code-build/ci/build_buildspec.yaml
+      privileged-mode: true
+      type: BUILD_GENERAL1_MEDIUM
+    - identifier: build_python_37_minimal_EXASOL_620
+      env:
+        variables:
+          FLAVOR: python-3.7-minimal-EXASOL-6.2.0
+      buildspec: ./aws-code-build/ci/build_buildspec.yaml
+      privileged-mode: true
+      type: BUILD_GENERAL1_MEDIUM
+    - identifier: build_python_38_minimal_EXASOL_620
+      env:
+        variables:
+          FLAVOR: python-3.8-minimal-EXASOL-6.2.0
+      buildspec: ./aws-code-build/ci/build_buildspec.yaml
+      privileged-mode: true
+      type: BUILD_GENERAL1_MEDIUM
```

Comment:
missing newline

Reply:
Needs to be fixed in the other project. I will fix it there....